src/main.js

```diff
@@ -0,0 +1,87 @@
+import Vue from 'vue'
+import App from './App.vue'
+import vuetify from './plugins/vuetify';
+import VueRouter from 'vue-router';
+import LoginPage from './views/LoginPage';
+import RegisterPage from './views/RegisterPage';
+import AdminPage from './views/AdminPage';
+import HomePage from './views/HomePage';
+import IssueBook from './views/IssueBook';
+import AddBook from './views/AddBook';
+import CustomerDetails from './views/CustomerDetails';
+import AGgrid from './views/AGgrid';
+import store from './store/store'
+
+Vue.use(VueRouter);
+const routes = [
+  {
+    path: '/login', name: 'LoginPage', meta: {
```

Comment:
put this on separate lines like helps in readabily: 
{
    path: '/register', 
    name: 'RegisterPage', 
    meta: {
      requiresAuth: true
    }, 
    component: RegisterPage
  },
  
  cange everywhere